libcxx/include/__chrono/formatter.h

```diff
@@ -589,9 +590,14 @@ __format_chrono(const _Tp& __value,
     __sstr << __value;
   else {
     if constexpr (chrono::__is_duration<_Tp>::value) {
-      if (__value < __value.zero())
-        __sstr << _CharT('-');
-      __formatter::__format_chrono_using_chrono_specs(__sstr, chrono::abs(__value), __chrono_specs);
+      if constexpr (numeric_limits<typename _Tp::rep>::is_signed) {
```

Comment:
I would use `std::is_signed_v<typename _Tp::rep>` instead. It allows you to include only `<__type_traits/is_signed.h>`.

Reply:
Actually there is a difference between the two. Users may specialize `numeric_limits`, but specializing `is_signed` is not allowed. Durations can use a user defined arithmetic type. So using  `numeric_limits` seems to be the better choice. I've added comment in the code.